plugin.js

```diff
@@ -28,7 +30,7 @@ function plugin (fn, options = {}) {
   }
 
   if (!options.name) {
-    options.name = pluginName
+    options.name = pluginName + count++
```

Comment:
```suggestion
    options.name = 'auto-loaded-' + pluginName + count++
```

Reply:
The plugin is auto-named, not auto-loaded, would be ok anyway? May I just add a -auto-42 at the end instead?

Reply:
That would be good. Just something to differentiate them from manually loaded plugins.